front/pages/api/w/[wId]/invitations/[iId]/index.ts

```diff
@@ -1,22 +1,26 @@
 import type {
+  ActiveRoleType,
   MembershipInvitationType,
   WithAPIErrorReponse,
 } from "@dust-tt/types";
+import { ActiveRoleSchema } from "@dust-tt/types";
 import { isLeft } from "fp-ts/lib/Either";
 import * as t from "io-ts";
 import * as reporter from "io-ts-reporters";
 import type { NextApiRequest, NextApiResponse } from "next";
 
 import { getInvitation, updateInvitationStatus } from "@app/lib/api/invitation";
 import { Authenticator, getSession } from "@app/lib/auth";
+import { MembershipInvitation } from "@app/lib/models/workspace";
 import { apiError, withLogging } from "@app/logger/withlogging";
 
 export type PostMemberInvitationsResponseBody = {
   invitation: MembershipInvitationType;
 };
 
 export const PostMemberInvitationBodySchema = t.type({
-  status: t.literal("revoked"),
+  status: t.union([t.literal("revoked"), t.undefined]),
```

Comment:
Let's require both that way we update everything always will make the code a bit simpler?